python/lsst/obs/subaru/gen3/hsc/instrument.py

```diff
@@ -93,6 +92,7 @@ def register(self, registry):
             )
 
     def getRawFormatter(self, dataId):
+        from .rawFormatter import HyperSuprimeCamRawFormatter, HyperSuprimeCamCornerRawFormatter
```

Comment:
Please add short docstring, and optionally the same explainer for this import as in the commit message.

Reply:
The docstrings for these methods are defined in the ABC parent class.